server-plugins/gmail-resources/src/index.ts

```diff
@@ -72,41 +71,43 @@ export async function OnMessageCreate (tx: Tx, control: TriggerControl): Promise
       res.push(tx)
     }
     if (message.incoming) {
-      const docs = await control.findAll(notification.class.DocUpdates, {
+      const docs = await control.findAll(notification.class.DocNotifyContext, {
         attachedTo: channel._id,
         user: message.modifiedBy
       })
       for (const doc of docs) {
+        // TODO: push inbox notification
```

Comment:
Should we fix TODO? Or in next PR?

Reply:
I'll try to fix in this pr, I need to wait for Denis and find out from him how to set up gmail locally
if it works, then for the telegram there will also be changes but without the test, since they have very similar code